chpl/chpl-api/e2e/collections/certified-product-controller.postman_collection.json

```diff
@@ -1267,6 +1267,184 @@
+						},
+						"url": {
+							"raw": "{{url}}/rest/certified_products/10306",
+							"host": [
+								"{{url}}"
+							],
+							"path": [
+								"rest",
+								"certified_products",
+								"10306"
+							]
+						},
+						"description": "Adding removed Test Tools to 2015 listing should give 400 status code and error in response"
+					},
+					"response": []
+				},
+				{
+					"name": "PUT /{certifiedProductId} - 400 status and error for adding new/invalid Test Tools for 2014 listing",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"exec": [
+									"var ReqBody= {\r",
+									"        userName:pm.environment.get(\"usernameONC\"),password:pm.environment.get(\"passwordONC\")\r",
+									"    };\r",
+									"pm.sendRequest({\r",
+									"    url: pm.environment.get(\"authurl\"),\r",
+									"    method: \"POST\",\r",
+									"    header: {\r",
+									"        \"Content-Type\": \"application/json\",\r",
+									"        \"API-KEY\": pm.environment.get(\"apiKey\")\r",
+									"    },\r",
+									"    body: JSON.stringify(ReqBody)\r",
+									"}, function (err, response) {\r",
+									"   var jsonData = response.json();\r",
+									"   pm.environment.set(\"token\",jsonData[\"token\"])\r",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Adding new Test Tools to 2014 listing should give 400 status code and error in response\", function () {\r",
+									"    pm.response.to.have.status(400);\r",
+									"    pm.expect(pm.response.text()).to.include(\"Test Tool 'NIST General Validation Tool (GVT)' is not valid\");\r",
+									"    pm.expect(pm.response.text()).to.include(\"Test Tool 'NCPDP Electronic Prescribing (eRx) Testing Tool' is not valid\");\r",
+									"});\r",
+									"pm.test(\"Adding invalid Test Tool to listing should give 400 status code and error in response\", function () {\r",
+									"    pm.response.to.have.status(400);\r",
+									"    pm.expect(pm.response.text()).to.include(\"contains an invalid test tool 'HL7 CDA Cancer Registry Reporting Validation Tool'\");\r",
+									"});"
```

Comment:
This test set exactly matches to that on line 1310-1312 above (the one for 2015 listing), and I think therefore the plan is treating it as a duplicate due to test string match and running it only once (for 2015 listing as it's first in order) and this one for 2014 does not run. I'd suggest to review that on Bamboo and if true, update test string to be unique.

I also think that status check under this test(line 1398) does not add value as the test above (line 1393) checks that and if that fails, this would fail too and vice versa.

Reply:
Updated test string and removed the status check . Pushed changes for this